feature/inform/src/main/java/com/school_of_company/inform/util/DateFormatter.kt

```
@@ -0,0 +1,20 @@
+
+fun formatIsoWithoutTimezone(isoDate: String): String {
+    return try {
+        val inputFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.getDefault())
+        inputFormat.timeZone = TimeZone.getDefault()
+
+        val outputFormat = SimpleDateFormat("yyyy.MM.dd / HH:mm", Locale.getDefault())
+        outputFormat.timeZone = TimeZone.getDefault()
+
+        val date = inputFormat.parse(isoDate)
+        date?.let { outputFormat.format(it) } ?: ""
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)